app-admin/lsat/lsat-0.9.8.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="The Linux Security Auditing Tool"
+HOMEPAGE="http://usat.sourceforge.net"
+SRC_URI="https://github.com/triode3/lsat/archive/v${PV}.tar.gz"
```

Comment:
Append `-> ${P}.tar.gz` to satisfy CI bot. As you can see, the distfile isn't recognizable as-is.

Reply:
Oops!